src/SignalRServiceExtension/Bindings/JTokenExtensions.cs

```diff
@@ -1,17 +1,23 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
 {
     internal static class JTokenExtensions
     {
+        public static readonly JsonSerializer JsonSerializer = JsonSerializer.Create(new JsonSerializerSettings
```

Comment:
How about move all the customize JsonConverter to `ConfigProvider` as global settings, so you don't have to config it specifically. 

Something like this: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Config/WebPubSubConfigProvider.cs#L150

Reply:
I did it the same before... but reported from customer that this setting overrides the customer's setting. So I have made this PR. See #229